winrt/lib/xaml/BaseControl.h

```diff
@@ -788,7 +792,18 @@ namespace ABI { namespace Microsoft { namespace Graphics { namespace Canvas { na
             return ExceptionBoundary(
                 [&]
                 {
-                    if (++m_loadedCount == 1)
+                    boolean should_load = false;
```

Comment:
For us, `Loaded` or `Unloaded` messages don't have a clear meaning now. They mean, something may have changed in the state of the native tree. So, let's have a private method `UpdateLoadedState` that gets called from both `Loaded` and `Unloaded`.

Reply:
They still have the same meaning. So from the CanvasControl's point of view it is still Loaded and Unloaded.